torch/csrc/distributed/c10d/init.cpp

```diff
@@ -566,8 +571,7 @@ They are used in specifying strategies for reduction collectives, e.g.,
       .value("BAND", ::c10d::ReduceOp::RedOpType::BAND)
       .value("BOR", ::c10d::ReduceOp::RedOpType::BOR)
       .value("BXOR", ::c10d::ReduceOp::RedOpType::BXOR)
-      .value("PREMUL_SUM", ::c10d::ReduceOp::RedOpType::PREMUL_SUM)
-      .export_values();
+      .value("PREMUL_SUM", ::c10d::ReduceOp::RedOpType::PREMUL_SUM);
```

Comment:
Can we just make pybind API on `ReduceOp` class directly, so that user can still do things like `ReduceOp.SUM` (i.e. as a class attribute)?

I feel maybe we can do sth like this `ReduceOp.PREMUL_SUM(premul_value)` looks cleaner than `_make_nccl_premul_sum`